src/gwt/src/org/rstudio/studio/client/workbench/views/environment/dataimport/DataImportOptionsUiCsvLocale.ui.xml

```diff
@@ -0,0 +1,86 @@
+<ui:UiBinder xmlns:ui='urn:ui:com.google.gwt.uibinder'
+             xmlns:g='urn:import:com.google.gwt.user.client.ui'>
+
+   <ui:style type="org.rstudio.studio.client.workbench.views.environment.dataimport.DataImportOptionsUiCsvLocale.DataImportOptionsUiCsvLocaleStyle">
+      .dialog {
+         width: 420px;
+      }
+      .textBox {
+         width: 90px;
+      }
+      table.localeTable {
+        width: 100%;
+
+        padding-left: 5px;
+        padding-right: 5px;
+      }
+      td.checkbocCell {
+        text-align: right;
+        padding-right: 8px;
+      }
+      td.spacedCell {
+        padding-left: 10px;
+      }
+   </ui:style>
+
+   <g:HTMLPanel styleName="{style.dialog}">
+      <table class="{style.localeTable}">
+         <tr>
+            <td>
+                <g:Label text="Date Name:" />
+            </td>
+            <td>
+                <g:TextBox ui:field="dateName_" styleName="{style.textBox}" text="en" />
+            </td>
+            <td class="{style.spacedCell}">
+                <g:Label text="Encoding:" />
+            </td>
+            <td>
+                <g:ListBox ui:field="encoding_" />
+            </td>
+         </tr>
+         <tr>
+            <td>
+                <g:Label text="Date Format:" />
+            </td>
+            <td>
+                <g:TextBox ui:field="dateFormat_" styleName="{style.textBox}" text="%AD" />
```

Comment:
Could we add a link to the documentation for date/time formats in readr (perhaps via redirect) so that it's clear how to construct this format string?